src/components/Article.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Article({
+  title,
+  date,
+  text,
+}) {
+  return (
+    <>
+      <h1>{title}</h1>
+      <span>{date}</span>
+      <p>{text}</p>
+    </>
```

Comment:
Better to use semantic tag
```suggestion
    <article>
      <h1>{title}</h1>
      <span>{date}</span>
      <p>{text}</p>
    <acticle/>
```